tensorflow/stream_executor/cuda/cuda_driver.cc

```diff
@@ -38,6 +38,14 @@ limitations under the License.
 #include "tensorflow/stream_executor/platform/port.h"
 #include "tensorflow/stream_executor/lib/inlined_vector.h"
 
+#if defined(PLATFORM_WINDOWS)
+// FIXME: in windows ARRAYSIZE is defined in winnt.h but including it
```

Comment:
Nit: The string `FIXME` gets intercepted by one of the linters we use internally. Can you please change it to `TODO`? Thanks!